apps/frontend/core-dapp/src/features/deposit/DepositWarning.tsx

```diff
@@ -0,0 +1,109 @@
+import { observer } from 'mobx-react-lite'
+import { Alert as BaseAlert, Icon } from 'prepo-ui/lib/src'
+import styled from 'styled-components'
+import { spacingIncrement } from 'prepo-ui'
```

Comment:
I'm not sure why the build is failing (not recognizing the `prepo-ui/lib/src` path on vercel but works locally, could you try importing from prepo-ui instead of `lib/src`

```suggestion
import styled from 'styled-components'
import { Alert as BaseAlert, Icon, spacingIncrement } from 'prepo-ui'
```

Reply:
Good catch! That seems to be it. It was an auto-import from the IDE, maybe I need to restrict imports from `/lib/src`